web/src/frontend/classic/stores/Helpers.ts

```diff
@@ -0,0 +1,47 @@
+import { writable } from 'svelte/store';
+import type { IValue, Setting} from '../../../engine/SettingsManager';
+import type { Key } from '../../../engine/SettingsGlobal';
+
+const globalsettings = HakuNeko.SettingsManager.OpenScope();
+
+/**
+ * Create a writable svelte store that is coupled with the given setting
+ * and updated whenever the value of the underlying setting is changed.
+ * @param setting - a specific setting of the frontend
+ */
+export function CreateWritableStore<T extends IValue>(setting:Setting<T>) {
```

Comment:
Consider to make method generic and add type of the setting e.g., `Create<V extends IValue, S extends Setting<V>>(setting: S)`